llvm/lib/Target/AMDGPU/AMDGPUAttributor.cpp

```diff
@@ -1064,6 +1064,17 @@ static bool runImpl(Module &M, AnalysisGetter &AG, TargetMachine &TM) {
     } else if (CC == CallingConv::AMDGPU_KERNEL) {
       addPreloadKernArgHint(F, TM);
     }
+
+    for (auto &I : instructions(F)) {
+      if (auto *LI = dyn_cast<LoadInst>(&I)) {
```

Comment:
`AAAddressSpace` for now only supports load and store instructions. Will add more later.